src/main/java/com/google/codeu/servlets/ReviewServlet.java

```diff
@@ -0,0 +1,53 @@
+package com.google.codeu.servlets;
+
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Location;
+
+import com.google.appengine.api.datastore.*;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.List;
+
+@WebServlet("/review")
+public class ReviewServlet extends HttpServlet {
+
+    private Datastore datastore;
+
+    @Override
+    public void init() {
+        datastore = new Datastore();
+    }
+
+
+
+    @Override
+    protected void doPost(HttpServletRequest request,
+                          HttpServletResponse response) throws ServletException, IOException {
+
+        // location being reviewed
+        Entity location = datastore.getLocation(""); // Argument to be changed upon creation of Location Feed
+        List<Location.Review> reviews = (List<Location.Review>) location.getProperty("reviews");
```

Comment:
Does this work? Firstly, I know that properties can be collections, but I don't know if they're guaranteed to be `List`. (Maybe `Collection` would work.) Secondly, can they be collections of `Location.Review`?